__tests__/app.test.js

```diff
@@ -39,5 +93,22 @@ describe('Error handling', () => {
          expect(body).toEqual({"message" : "Invalid path provided - please try again"});
         });
     });
-});
 
+    test('404: GET followed by invalid review datatype should return a message', () => {
+        return request(app)
+        .get("/api/reviews/bananas")
+        .expect(400)
+        .then(({body})=>{
+            expect(body.message).toEqual("Bad Request")
+        })
+    });
+
+    test('400: GET follow by invalid review id ie resource that doesn\'t exist should return a message', () => {
+        return request(app)
+        .get("/api/reviews/99999")
+        .expect(404)
+        .then(({body})=>{
+            expect(body.message).toEqual("id does not exist")
+        });
+    });
```

Comment:
Think about where this error is going - to a user. Perhaps 'article does not exist' would make more sense.